### PR TITLE
Integrate Osano consent manager

### PR DESCRIPTION
**Pull request by viphan007**
**Website Team Ticket:** https://github.com/Consensys/dx-team/issues/1518

### CODE REVIEW

**Review by jlazoff — APPROVED** [review]
Legal Consent Manager

**Comment by jlazoff** [comment]
Hi @davidmurdoch @kevinbluer ,

Hope all is well. 

Can you please approve this PR? Osano is a legal compliance data privacy management platform, being implemented across CSI. Link: https://www.osano.com/ 

Appreciate your help,

Joshua

File: custom_theme/main.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {% extends "base.html" %}
-
-{% block site_meta %}
+  
+  {% block site_meta %}
+    <script src="https://cmp.osano.com/AzZMxHTbQDOQD8c1J/a2e89f0e-f467-4542-bfea-30ea2c1a6648/osano.js"></script>
```
Comment (cliffoo) [inline custom_theme/main.html:4]:
```suggestion
{% block site_meta %}
    <script src="https://cmp.osano.com/AzZMxHTbQDOQD8c1J/a2e89f0e-f467-4542-bfea-30ea2c1a6648/osano.js" defer></script>
```

Comment (jlazoff) [reply custom_theme/main.html:4]: Removing "defer"

`Important: The Osano.js should be placed without any async or defer attributes. Compliance cannot be guaranteed when using the above attributes. `

**Link:** https://docs.osano.com/installing-osano

**Comment by jlazoff** [comment]
Hi, 

This Legal widget is very important. Please confirm that: 

- [ ] Osano script placement, see below:
### **Important: Osano will only be able to discover and block scripts that are loaded after it, therefore it must execute in the <head> before any other 3rd party or tracking content.** 

- [x] I have removed "defer" 
### _**Important: The Osano.js should be placed without any async or defer attributes. Compliance cannot be guaranteed when using the above attributes.**_ 

Details Link: https://docs.osano.com/installing-osano

**Review by cliffoo — APPROVED** [review]

**Pull request merged**